website/src/app/docs/reference/env-vars/readme.mdx

```diff
@@ -121,11 +122,13 @@ All secrets should be a **base64-encoded string**.
 
 ### WireGuard
 
-| Env Key                | Description                                                     | Format  | Default |
```

Comment:
This is the autogenerated file so will be overridden on next docs build.

Reply:
I didn't update the docs build to be compatible with the new website/docs structure since it would require quite a bit of work. The new docs don't support YAML front matter and use inline components for `<Alert>` from Flowbite.

I lean towards the opinion it may not be worth refactoring the docs writer for 0.7, instead just making the changes we need directly.

What do you think?

Reply:
Alternatively the best approach might be to write `JSON` from the elixir side and then choose how to parse and display these in the website -- might be more maintainable in the long run?

Reply:
I think it's not worth refactoring for 0.7 but definitely will be worth it for 1.0. Let's just remember to never re-generate docs in this branch again then.

Reply:
SGTM